manage.py

```diff
@@ -89,7 +89,7 @@ def setup_dev():
                   .filter_by(permissions=Permission.AGENCY_WORKER)
                   .first(),
                   confirmed=True)
-    worker.agency = Agency.query.filter_by(name='SEPTA').first()
+    worker.agencies = [Agency.query.filter_by(name='SEPTA').first()]
```

Comment:
I think this is okay. We are always going to have a SEPTA object in the db when we get here, and we only have this for debugging locally. This is the default agency worker^